snmp/datadog_checks/snmp/data/profiles/generic-router.yaml

```diff
@@ -9,4 +9,7 @@ extends:
   - _generic-udp.yaml
   - _generic-ospf.yaml
 
+device:
+  vendor: "generic"
```

Comment:
isnt it better to not include a tag in this case?

Reply:
Yes, I think you are right. Let's remove it.